README.md

```diff
@@ -18,40 +18,74 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True, this is because it is comparing the values so even though one is a string it is coercing the value to be true.
+
 **b)** `"1" === 1`
 
+False, because when using triple equals you are comparing the pure values and a string does not equal the number.
+
 **c)** `x == 'x'`
 
+When x is undefined it cannot equal the other
+
```

Comment:
More specifically, the reason this is false is because we are comparing variable x to string x. Variable x could contain any value, but unless the value is exactly 'x' it will never == the right side.